### PR TITLE
CB-244: Rating System (Docs)

### PR DESCRIPTION
**Pull request by ps2611**

### CODE REVIEW

**Comment by ps2611** [comment]
Changed text and rating rows from bold to simple fonts.

File: critiquebrainz/ws/review/views.py [hunk 1]
```diff
@@ -225,7 +228,13 @@ def review_modify_handler(review_id, user):
 
     **OAuth scope:** review
 
+    :json string text: Text part of review, min length is 25, max is 5000 **(optional)**
+    :json integer rating: Rating part of review, min is 1, max is 5 **(optional)**
+
+    **NOTE:** The value of unmodified parameter should be set to its value in previous revison.
+
     :statuscode 200: success
+    :statuscode 400: invalid request (see source)
```
Comment (gentlecat) [inline critiquebrainz/ws/review/views.py]: Nope. No one is going to go and see the source.

File: critiquebrainz/ws/review/views.py [hunk 2]
```diff
@@ -225,7 +228,13 @@ def review_modify_handler(review_id, user):
 
     **OAuth scope:** review
 
+    :json string text: Text part of review, min length is 25, max is 5000 **(optional)**
+    :json integer rating: Rating part of review, min is 1, max is 5 **(optional)**
+
+    **NOTE:** The value of unmodified parameter should be set to its value in previous revison.
```
Comment (gentlecat) [inline critiquebrainz/ws/review/views.py]: Didn't we decide that it's not going to be like that?

Comment (ps2611) [reply critiquebrainz/ws/review/views.py]: Yes. I will update it once you confirm these changes https://github.com/metabrainz/critiquebrainz/pull/154/commits/2b1eb3738b80657ffc1fc4dc15c1c7cc16ba8f76

File: critiquebrainz/ws/review/views.py (hunk 1 above)
Comment (ps2611) [reply critiquebrainz/ws/review/views.py]: Ok. Will remove it.

**Comment by ps2611** [comment]
>  It would be fine to put changes that you make here into the PR that actually changes the implementation that is being documented.

Yes, its better. I will move the ws docs part in #154.

**Comment by gentlecat** [comment]
OK, I'm going to close this PR. Don't forget to fix the issues that I mentioned before.

**Pull request closed without merging**